blocks/copilot/pre-generated.js

```diff
@@ -0,0 +1,10 @@
+const pregenerateSequence = [
+  `
+    <img loading="lazy" alt="" type="image/jpeg" src=https://eisdevelopmentgroupva7di.blob.core.windows.net/firefall-asset-dev/5c01708c-8c41-4e79-86f5-74c407445536.jpeg?se=2023-11-21T13%3A00%3A16Z&sp=r&sv=2022-11-02&sr=b&sig=aj%2BH4MrVM83x2YfQPI43CmuMnixxxl93%2ByPqCF2En3Q%3D width="200" height="300">
+        <h1>Indian cuisine is diverse, flavorful, and often vegetarian. Spices and herbs are used extensively, and regional specialties abound.</h1>
+        <hr>
+    `,
+];
+
+// eslint-disable-next-line import/prefer-default-export
+export { pregenerateSequence };
```

Comment:
Where is this function `pregenerateSequence` used?

Reply:
Not yet. I added this to keep pre-generated responses for demo.